lib/github.ml

```diff
@@ -137,3 +139,19 @@ let gh_link_of_string url_str =
       with _ -> None
     end
   | _ | (exception Re2.Exceptions.Regex_match_failed _) -> None
+
+let get_project_owners (labels : label list) project_owners_map =
+  List.fold_left labels ~init:[] ~f:(fun acc label ->
+    match Map.find_multi project_owners_map label.name with
```

Comment:
Similar to other comment, I wonder if we should handle the case where we can't find label (as they can be removed etc)